news/cross.rst

```diff
@@ -0,0 +1,23 @@
+**Added:**
+
+* Added partial support for cross compiling (Unixes can compile for other unixes only)
```

Comment:
Would suggest saying Linuxes here since it sounds like we are not trying to handle FreeBSD, etc.

```suggestion
* Added partial support for cross compiling (Linuxes can compile for other linuxes only)
```

Reply:
I meant Linux and OSX. There's nothing `FreeBSD` specific here and once `conda` supports it, there's nothing to change here.

Reply:
Oh interesting. So this works with macOS? Would have assumed we need more things there like frameworks and such.

Reply:
No it doesn't. Other things like `conda-forge-ci-setup` need to be fixed first